pkg/extender/extender.go

```diff
@@ -273,14 +273,25 @@ func (e *Extender) getNodeScore(
 	regionInfo *localityInfo,
 	idMap map[string]*volume.NodeInfo,
 ) int {
+	logger := log.WithFields(log.Fields{
+		"node": node.Name,
+	})
+	logger.Debugf("getNodeScore, let's go")
 	for _, address := range node.Status.Addresses {
 		if address.Type != v1.NodeHostName {
 			continue
 		}
+		logger.Debugf("rack info: %+v", rackInfo)
+		logger.Debugf("zone info: %+v", zoneInfo)
+		logger.Debugf("region info: %+v", regionInfo)
 		nodeRack := rackInfo.HostnameMap[address.Address]
 		nodeZone := zoneInfo.HostnameMap[address.Address]
 		nodeRegion := regionInfo.HostnameMap[address.Address]
 
+		logger.Debugf("nodeRack: %v", nodeRack)
+		logger.Debugf("nodeZone: %v", nodeZone)
+		logger.Debugf("nodeRegion: %v", nodeRegion)
```

Comment:
I think this is too much info to print in each loop even for Debug.

Also, all logs in the context of a filter or prioritize call should be printed with PodLog so that we can correlate the logs to a pod